pine64/pinebook-pro/README.md

```diff
@@ -0,0 +1,98 @@
+> **NOTE**: This is an attempt to port [samueldr](https://github.com/samueldr/)'s [wip-pinebookpro](https://github.com/samueldr/wip-pinebook-pro) to [nixos-hardware](https://github.com/NixOS/nixos-hardware). Credit for the work done goes to the original authors.
+
+## Using in your configuration
+
+To use this configuration simply add this module to your configuration.
+
+```nix
+{
+  imports = [
+    ${nixos-hardware}/pine64/pinebook-pro
+  ];
+}
+```
+
+That entry point will try to maximize the hardware compatibility.
+
+## Current state
+
+Currently everything seems to work but this is not guaranteed it will be forever.
+
+### Kernel
+
+By default this uses the latest upstream kernel but the overlay makes available `pinebookpro-unstable-manjaro-kernel`
+which is built from a pinned kernel version and makes available the DP alt mode over Type-C.
+This kernel is unstable and some sporadic kernel panics may occur, use at your own risk.
```

Comment:
```suggestion
This module defaults to the latest upstream kernel. Attempts have been made to use manjaro's kernel that makes available the DP alt mode over Type-C, but it turns out that the kernel is unstable and some sporadic kernel panics may occur. If you want to test version, you can grab the nix expression from this commit: https://github.com/NixOS/nixos-hardware/blob/6d1bd5bc2e8b9992a3f57e416ba50fbed5516db6/pine64/pinebook-pro/kernel/default.nix
```

Reply:
I didn't know that is possible to link commits from forks as they were in the forked repo.
Then, I suppose that we can't squash all the commits into a single one before merging,

Reply:
The commit will persisted by github, even if you squash it.